common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1536,6 +1536,7 @@ public static final class BrokerContentProvider {
         public enum API {
             MSAL_HELLO(MSAL_HELLO_PATH, null, VERSION_3),
             ACQUIRE_TOKEN_INTERACTIVE(MSAL_ACQUIRE_TOKEN_INTERACTIVE_PATH, null, VERSION_3),
+            ACQUIRE_TOKEN_ATV2_INTERACTIVE(MSAL_ACQUIRE_TOKEN_ATV2_INTERACTIVE_PATH, null, VERSION_3),
```

Comment:
>VERSION_3 [](http://example.com/codeflow?start=91&length=9)

Do we need to update this?

Reply:
I'm honestly not sure, was hoping someone more familiar with this part of the code would give a definitive answer 😅

Reply:
I'm pretty sure this needs to be updated because older brokers are not going to understand ATv2 API.

Reply:
yes, just that this parameter is not in used today but can be put to use when making call from BrokerMsalController

Reply:
Is this not already covered by bumping the broker protocol version to 16?

Reply:
You may update here as well (don't see any effect of it though)